homeassistant/components/vesync/switch.py

```diff
@@ -4,11 +4,12 @@
 
 from homeassistant.components.switch import SwitchEntity
 from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
-from .common import VeSyncDevice
+from .common import DEVICE_HELPER, VeSyncBaseEntity, VeSyncDevice
```

Comment:
In order to split this PR, you should be able to add support for humidifier switches in a preliminary PR.

Reply:
Same as the sensors, humidifiers aren't detected as devices right now. But, I can split this change out and update switches after the humidifier support exists